eng/versioning/version_client.txt

```diff
@@ -151,6 +151,8 @@ com.azure.spring:azure-spring-boot-test-aad-resource-server-by-filter;1.0.0;1.0.
 com.azure.spring:azure-spring-boot-test-core;1.0.0;1.0.0
 com.azure.spring:azure-spring-boot-test-cosmosdb;1.0.0;1.0.0
 com.azure.spring:azure-spring-boot-test-keyvault;1.0.0;1.0.0
+com.azure.spring:azure-spring-boot-test-keyvault-reactive;1.0.0;1.0.0
```

Comment:
will the version scripts take care of `pom-reactive.xml`

Reply:
Good question. 👍 
Now it won't. I'll add a commit to fix it.

Reply:
Added.